_includes/resources.md

```diff
@@ -1,13 +1,8 @@
-### General Information
+Aeon is developed from a community-driven approach, so there are no official resources or official organizations that are leading it. Smooth, a developer for AEON since 2014, has maintained regular updates to the source code on [GitHub](https://github.com/aeonix/aeon). Additional support is provided by community developer Stoffu as well as many community contributors. Everyone is invited to contribute to the AEON project whether that be developing new applications or integrating AEON as a payment method. 
```

Comment:
This file replaces **official** resources. 

`Aeon is developed from a community-driven approach, so there are no official resources or official organizations that are leading it.`

Surely, you must agree with this statement.